src/releasers/go-yoshi.ts

```diff
@@ -0,0 +1,151 @@
+// * we need a deny list of modules that are mono repos, which we keep track of
```

Comment:
Not sure what's going on with the comment block here.  At the very least, this should be below the copyright.